src/blocks/vpn/mullvad.rs

```diff
@@ -0,0 +1,78 @@
+use regex::Regex;
+use std::process::Stdio;
+use tokio::process::Command;
+
+use crate::blocks::prelude::*;
+use crate::util::country_flag_from_iso_code;
+
+use super::{Driver, Status};
+
+pub struct MullvadDriver {
+    regex_country_code: Regex,
+}
+
+impl MullvadDriver {
+    pub async fn new() -> MullvadDriver {
+        MullvadDriver {
+            regex_country_code: Regex::new("Connected to ([a-z]{2}).*, ([A-Z][a-z]*).*\n").unwrap(),
+        }
+    }
+
+    async fn run_network_command(arg: &str) -> Result<()> {
+        Command::new("mullvad")
+            .args([arg])
+            .stdin(Stdio::null())
+            .stdout(Stdio::null())
+            .spawn()
+            .error(format!("Problem running mullvad command: {arg}"))?
+            .wait()
+            .await
+            .error(format!("Problem running mullvad command: {arg}"))?;
+        Ok(())
```

Comment:
Should we also check the exit status?

Reply:
Well, we _can_ check it, but it is not necessary, because in case of errors the `status` string will not match any variants except  `Error`.

I should probably add a check for nonzero `ExitStatus`. I'll implement it today.

Reply:
More elaborate checks are useless because `mullvad` exits with code `1` on all errors except panics (they result in `101` code).

Reply:
Yeah, I meant `ExitStatus::success()`.